lib/tests/streamlit/runtime/caching/hashing_test.py

```diff
@@ -291,6 +291,23 @@ class EnumClass(Enum):
 
         assert get_hash(enum_1)
 
+    def test_different_enums(self):
+        class EnumClassA(Enum):
+            ENUM_1 = "hello"
+            ENUM_2 = "world"
+
+        class EnumClassB(Enum):
+            ENUM_1 = "hello"
+            ENUM_2 = "world"
+
+        enum_a = EnumClassA.ENUM_1
+        enum_b = EnumClassB.ENUM_1
+
+        assert get_hash(enum_a)
+        assert get_hash(enum_b)
```

Comment:
This looks a little strange to me - are we just asserting that `get_hash` does not return 0 or None? The `get_hash` function always returns a string, so the assertion here is meaningless and should be removed, unless I'm misunderstanding something

Reply:
I think here and in other places, in this file, the idea of this is to make sure that this `get_hash` call will not raise `UnhashableTypeError` exception.

Reply:
Since we anyway implicitly check that by `get_hash` calls in our tests, I will remove the `assert`s

Reply:
Yeah - and I think we're on the same page, but just to clarify (for anyone else reading along): if an exception is raised in a unit test, that unit test will fail, so we can achieve "asserting an exception is not raised" by just calling the function we want to test.